public/docs/_examples/attribute-directives/dart/lib/app_component.dart

```diff
@@ -0,0 +1,13 @@
+// #docregion
+library attribute_directives.app_component;
```

Comment:
Gah, scratch that. I was getting "directive" confused with "decorator". Never mind...